ClassRegistration/ClassRegistration.App/Repositories/EnrollmentRepo.cs

```diff
@@ -0,0 +1,58 @@
+﻿using ClassRegistration.App.Interfaces;
+using ClassRegistration.DataAccess.Entity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ClassRegistration.App.Repositories
+{
+    public class EnrollmentRepo : IEnrollmentRepo
+    {
+        /// <summary>
+        /// A private field for the database entity
+        /// </summary>
+        private Course_registration_dbContext _dbContext;
+
+        /// <summary>
+        /// A constructor that intializes the database entity.
+        /// </summary>
+        /// <param name="dbcontext"></param>
+        public EnrollmentRepo(Course_registration_dbContext dbcontext)

```

Comment:
We'll be using the Generic Repository pattern, so this will change, but it's correct for now